pkg/git/models.go

```diff
@@ -0,0 +1,8 @@
+package git
+
+// Git struct holds necessary data to work with the current git repository
+type Git struct {
```

Comment:
This is somewhat of an anti pattern in go. Remove this type and put where it's used. Ie. top of the git.go file.

Reply:
Fixing also will fix the auto package as well.